dangerzone/conversion/errors.py

```diff
@@ -27,8 +27,8 @@ def get_subclasses(cls) -> List[Type["ConversionException"]]:
 class QubesNotEnoughRAMError(ConversionException):
```

Comment:
Might be worth changing the name of the error class as well?

Reply:
You're right. How does `QubesConversionStartFailed` sound?